pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -268,18 +274,19 @@ def evaluate(self, model, dataloaders, max_batches, test_mode: bool = False):
                 # -----------------
                 # RUN EVALUATION STEP
                 # -----------------
-                output = self.evaluation_forward(model, batch, batch_idx, dataloader_idx, test_mode)
+                output = self.evaluation_forward(model, batch, batch_idx, dataloader_idx)
```

Comment:
the problem in tests is here that in this time being (`test_trainer.test_benchmark_option`) the trainer mode is `TRAINING` which is not expected in for evaluation... can you check the flow before and fix it?